packages/core/src/FormField/FormField.tsx

```diff
@@ -1,10 +1,27 @@
 // @ts-nocheck
 
 import React from 'react'
+import styled from 'styled-components'
 import { Box, IBoxProps } from '../Box'
 import { IconField } from '../IconField'
+import { getPaletteColor } from '../utils'
 
-export interface IFormFieldProps extends IBoxProps {}
+export interface IFormFieldProps extends IBoxProps {
+  disabled?: boolean
```

Comment:
I think we also wanted to add styling for `readOnly` that looks the same as disabled https://stackoverflow.com/a/7730719

Reply:
I'll split that work into a separate PR as this one is already getting quite big, if you're alright with that?